lib/core/versions/latest/AnchorFile.ts

```diff
@@ -167,9 +167,9 @@ export default class AnchorFile {
 
```

Comment:
This PR missed at least a couple of snake_casing properties, `did_suffix` and `signed_data` for instance, please make sure all snake_casing properties are switched over.

Reply:
`map_file_uri`

Reply:
`writer_lock_id`

Reply:
`chunk_file_uri`